hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -118,12 +118,10 @@ def submit_content_request_from_s3_event_record(
     """
     Converts s3 event into a SubmitContentRequestBody object with a URL to the content
     """
-    # For partner buckets we use the full bucket name and key as the content ID to avoid collisions with
-    # existing objects
     bucket = record["bucket"]["name"]
     key = record["object"]["key"]
-    content_id = bucket + "/" + key
 
+    content_id = get_content_id_from_bucket_and_key(bucket, key)
     url = create_presigned_url(bucket, key, None, 3600, "get_object")
 
     return SubmitContentRequestBody(
```

Comment:
Would love to see the full S3 URL (s3://...) being added as an additional field. That way we are not losing much information in the process.